app/components/shared/Header/Header.jsx

```diff
@@ -77,7 +129,16 @@ class Header extends React.Component {
                 </span>
             )
 
-            if (this.props.tokenIdentity.ttl) {
+            if (this.state.tokenRenewed) {
```

Comment:
Little bit of a hacky solution, but wasn't sure how to update the TTL once renewed since the value came from a super class. I'm definitely open to suggestions if there are better ways to handle this.